impl/src/main/java/com/okta/sdk/impl/ds/DefaultDataStore.java

```diff
@@ -16,6 +16,9 @@
  */
 package com.okta.sdk.impl.ds;
 
+import com.google.common.base.Charsets;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.io.CharStreams;
```

Comment:
IIRC, Guava is an optional dependency and should not be used here (I could be wrong)

I like Guava, but IMHO, it shouldn't be used in libs as it has historically had many breaking changes